src/AdSettings.ts

```diff
@@ -53,5 +55,33 @@ export default {
    */
   setUrlPrefix(urlPrefix: string) {
     CTKAdSettingsManager.setUrlPrefix(urlPrefix);
+  },
+
+  /**
+   * Requests permission to track the user.
+   *
+   * Requires a [`NSUserTrackingUsageDescription`](https://developer.apple.com/documentation/bundleresources/information_property_list/nsusertrackingusagedescription) in your `Info.plist`
+   *
+   * @platform iOS 14
+   */
+  async requestTrackingPermission(): Promise<TrackingStatus> {
+    if (Platform.OS !== 'ios') return 'unavailable';
+    return await CTKAdSettingsManager.requestTrackingPermission();
+  },
+  /**
+   * Gets the current tracking status.
+   *
+   * @platform iOS 14
+   */
+  async getTrackingStatus(): Promise<TrackingStatus> {
+    if (Platform.OS !== 'ios') return 'unavailable';
+    return await CTKAdSettingsManager.getTrackingStatus();
+  },
+
+  /**
+   * Enable or disable the automatic Advertiser ID Collection. On iOS 14 it is recommended to only enable automatic Advertiser ID Collection when the user has granted permission to track. (@see `requestTrackingPermission()`)
+   */
+  setAdvertiserIDCollectionEnabled(enabled: boolean): void {
```

Comment:
yup, let's make sure this is documented in the README or so

Reply:
Alright, everything's documented in the README now